test/es-module/test-esm-loader.mjs

```diff
@@ -43,6 +43,15 @@ await assert.rejects(
   { code: 'ERR_INVALID_RETURN_PROPERTY_VALUE' },
 );
 
+try {
+  await import('esmHook/commonJsNullSource.mjs');
+} catch (e) {
+  assert.strictEqual(e.code, 'ERR_INVALID_RETURN_PROPERTY_VALUE');
+  assert(e.message.includes('"source"'));
+  assert(e.message.includes('"load"'));
+  assert(e.message.includes('got null'));
+}
```

Comment:
We prefer `match` in tests, because when it doesn’t match the two strings that were compared get printed; whereas asserting a boolean tells us nothing other than that the assertion fails.

```suggestion
} catch ({ code, message }) {
  assert.strictEqual(code, 'ERR_INVALID_RETURN_PROPERTY_VALUE');
  assert.match(/"source"/, message);
  assert.match(/"load"/, message);
  assert.match(/got null/, message);
}
```

Reply:
> because when it doesn’t match the two strings that were compared get printed; whereas asserting a boolean tells us nothing other than that the assertion fails

I've made the same review comment before; thanks for keeping me honest.